js/controller.js

```diff
@@ -500,12 +500,13 @@ module.exports = function(OO, _, $, W) {
     toggleAirPlayIcon: function(event) {
       if (!this.state.airPlayStatusIcon) {
         const airPlayState = window.sessionStorage.getItem('airPlayState');
-        if (airPlayState) {
-          this.state.airPlayStatusIcon = airPlayState;
-          this.state.mainVideoElement.webkitShowPlaybackTargetPicker();
-        } else {
-          this.state.airPlayStatusIcon = 'airPlay-disconnected';
+        if (airPlayState === 'airPlay-connected') {
+          setTimeout(() => {
```

Comment:
please add a comment why do you use setTimeout and why it's so big

Reply:
I thought it would be more userfriendly to show it with some delay after buffering. But it works without settimeout.